portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,6 +19,8 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```

Comment:
Doesn't look like anything really changed in this file, remove the unused imports and extra newline at the bottom.